cogs/games.py

```diff
@@ -33,10 +36,436 @@
 ]
 
 
+class connect4(menus.Menu):
```

Comment:
Classes should start with uppercase first letter.
```suggestion
class Connect4(menus.Menu):
```
(Note, this also means you'll need to change all references to it)